src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -185,6 +189,16 @@ class WorkspaceServiceSpec extends FreeSpec with ScalaFutures with ScalatestRout
       }
     }
 
+    "when calling PATCH on the workspaces/*/*/updateAttributes to add/update an attribute" - {
+      "OK response is returned" in {
+        (Patch(attributeUpdatePath, MockWorkspaceServer.mockUpdateAttributeOperation)
+        ~> dummyAuthHeaders
+        ~> sealRoute(routes)) ~> check {
+          status should equal(OK)
+        }
+      }
+    }
```

Comment:
Here you would add the negative case that would return what Rawls would return as an error case.